Analysis/tasks.py

```diff
@@ -0,0 +1,69 @@
+import law
+import luigi
+import os
+import shutil
+import time
+import yaml
+from RunKit.sh_tools import sh_call
+from RunKit.checkRootFile import checkRootFileSafe
+
+from run_tools.law_customizations import Task, HTCondorWorkflow, copy_param, get_param_value
+from AnaProd.tasks import AnaTupleTask
+
+class HistProducerTask(Task, HTCondorWorkflow, law.LocalWorkflow):
```

Comment:
I was thinking to set like HistProducerFile and HistProducerSample scripts as the the SkimEvents.py and SkimProducer.py ones. Because I call the HistProducerSample if I change something that has to be propagated in all histograms. So my idea is to have `HistProducerTask` (which produces for all samples), `HistMergeTask` and then plot that can also be a task